geoips/filenames/base_paths.py

```diff
@@ -1,252 +1,171 @@
+        "GEOIPS_VERS": os.getenv("GEOIPS_VERS", "0.0.0"),
+        # Operational User
+        "GEOIPS_OPERATIONAL_USER": os.getenv("GEOIPS_OPERATIONAL_USER", False),
+        # Copyright Information
+        "GEOIPS_COPYRIGHT": os.getenv("GEOIPS_COPYRIGHT", "NRL-Monterey"),
+        "GEOIPS_COPYRIGHT_ABBREVIATED": os.getenv(
+            "GEOIPS_COPYRIGHT_ABBREVIATED", "NRLMRY"
+        ),
+        # Configuration and Queue
+        "GEOIPS_RCFILE": os.getenv("GEOIPS_RCFILE", ""),
+        "DEFAULT_QUEUE": os.getenv("DEFAULT_QUEUE", None),
+        # Computer Identifier
+        "BOXNAME": socket.gethostname(),
+    }
+
+    paths.update(geoips_global_variables)
+
+    derivative_directory_paths = {
+        paths["GEOIPS_BASEDIR"]: {
+            "GEOIPS_TESTDATA_DIR": "test_data",
+            "GEOIPS_DEPENDENCIES_DIR": "geoips_dependencies",
+        },
+        paths["GEOIPS_OUTDIRS"]: {
+            # Data Output Paths
+            "PRESECTORED_DATA_PATH": "preprocessed/sectored",
+            "PREREAD_DATA_PATH": "preprocessed/unsectored",
+            "PREREGISTERED_DATA_PATH": "preprocessed/registered",
+            "PRECALCULATED_DATA_PATH": "preprocessed/algorithms",
+            "CLEAN_IMAGERY_PATH": "preprocessed/clean_imagery",
+            "ANNOTATED_IMAGERY_PATH": "preprocessed/annotated_imagery",
+            "FINAL_DATA_PATH": "preprocessed/final",
+            "PREGENERATED_GEOLOCATION_PATH": "preprocessed/geolocation",
+            # Scratch Directories
+            "SCRATCH": "scratch",
+            "LOCALSCRATCH": "scratch",
+            "SHAREDSCRATCH": "scratch",
+            # Log and Data Directories
+            "LOGDIR": "logs",
+            "GEOIPSDATA": "geoipsdata",
+            # Ancillary Data Directories
+            "GEOIPS_ANCILDAT_AUTOGEN": "ancildat_autogen",
+            "GEOIPS_ANCILDAT": "ancildat",
+            # WWW Paths
+            "TCWWW": "preprocessed/tcwww",
+            "PUBLICWWW": "preprocessed/publicwww",
+            "PRIVATEWWW": "preprocessed/privatewww",
+            # Tropical Cyclone Paths
+            "TC_DECKS_DB": "longterm_files/tc/tc_decks.db",
+            "TC_DECKS_DIR": "longterm_files/tc/decks",
+        },
+        paths["BASE_PATH"]: {
+            "TC_TEMPLATE": "plugins/yaml/sectors/dynamic/tc_web_template.yaml",
+        },
+    }
+
+    for top_directory, sub_directories in derivative_directory_paths.items():
+        for key, sub_path in sub_directories.items():
+            paths[key] = get_env_var(key, os.path.join(top_directory, sub_path))
+
+    if not os.getenv("HOME"):
+        # need home drive default for windows
+        paths["HOME"] = os.getenv("HOMEDRIVE") + os.getenv("HOMEPATH")
+    else:
+        paths["HOME"] = os.getenv("HOME").rstrip("/")
+
+    # Setting links for WWW Paths
+    www_paths = ["TCWWW", "PUBLICWWW", "PRIVATEWWW"]
+    for path in www_paths:
+        paths[f"{path}_URL"] = get_env_var(f"{path}_URL", paths[path])
+
+    return paths
```

Comment:
Do we need to update paths with www_paths?  Looks like only the URL version are in here now?

Reply:
I'm not sure what you mean here

Reply:
I update paths directly in this block, which is a bit different than how I did it above.

Reply:
```
    www_paths = ["TCWWW", "PUBLICWWW", "PRIVATEWWW"]
    for path in www_paths:
        paths[f"{path}_URL"] = get_env_var(f"{path}_URL", paths[path])
```
TCWWW, PUBLICWWW, and PRIVATEWWW are NOT in paths dictionary, correct?  They were before.

Reply:
They ARE in the paths dictionary, at least from what I'm seeing

Above, I set them to:

```
            # WWW Paths
            "TCWWW": "preprocessed/tcwww",
            "PUBLICWWW": "preprocessed/publicwww",
            "PRIVATEWWW": "preprocessed/privatewww",
```

And then in the for loop you shared I set them to the same value (which I _believe_ was what was happening in the og code)

Reply:
Oh, nevermind, I didn't see them up ther.e